CHANGELOG.md

```diff
@@ -8,15 +8,13 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ## [1.16.0] - 2016-09-22
 ### Added
-- New rule [`no-dynamic-require`]: restrict deep package imports to specific folders. ([#567])
```

Comment:
Yep, me neither. I knew it was weird that I didn't get any comment, I always mess up somewhere. Out of the 75+ rules I wrote, there's not even a handful that were merged without a comment :p